### ISSUE TITLE
Update the build forms recipe to use actions instead.

### ISSUE THREAD

**Issue by Abdo-reda** [withastro/docs#10374]
### 📚 Subject area/topic

HTML forms, and Actions

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/recipes/build-forms/

### 📋 Description of content that is out-of-date or incorrect

This recipe uses standard handling of html. A section should be included to use actions instead because that's the updated way and a more cleaner way of handling forms in astro.

**Comment by sarah11918** [comment]
Hi! I think we would welcome an update to have a recipe building forms with Actions!

Since there may be people who, for whatever reason, might prefer not to use actions, perhaps we can update this recipe to show the exact same thing using actions, and also keep the original at the bottom as an alternative. (Also, we need to update the prerequisite for how server-rendering is now handled!) 

Maybe something like:

```
# Build HTML forms in Astro Pages

With a server adapter, Astro pages can both display and handle forms. In this recipe, you’ll use Astro actions to submit form data to the server and check for errors.

<ReadMore>See how to use a standard HTML form without actions to submit data to the server. Your frontmatter script will handle the data on the server, sending no JavaScript to the client.</ReadMore>

Prerequisites
A project with [a server adapter](/en/guides/on-demand-rendering/#server-adapters) installed.
```

I will add `help wanted` on this issue for anyone who would like to update the recipe! 🙌

**Comment by Abdo-reda** [comment]
yes I completely agree, including both approaches would be nice.

**Comment by casungo** [comment]
I can take it! Will have a PR prob tomorrow
